file-native/src/main/java/org/ballerinalang/stdlib/file/utils/FileConstants.java

```diff
@@ -35,17 +35,23 @@
 
     /**
      * Package name.
+     * @deprecated Use ModuleUtils.getModule().getName()
      */
+    @Deprecated
     public static final String PACKAGE_NAME = "file";
 
     /**
      * Package version.
+     * @deprecated Use ModuleUtils.getModule().getVersion()
      */
+    @Deprecated
     public static final String PACKAGE_VERSION = "0.6.3";
```

Comment:
Anyone using this? If not we can remove this.

Reply:
done